### PR TITLE
 [guilib][depends] Remove libgif from core.

### PR DESCRIPTION
**Pull request by ace20022**
It's still needed for TexturePacker.

### CODE REVIEW

**Comment by ace20022** [comment]
@stefansaraev Not sure if this is enough/correct for texturepacker, could you please have a look?

**Comment by stefansaraev** [comment]
I am quite sure giflib for target can be removed. just keep giflib-native as-is.

and perhaps you should not docs/README.* - giflib is still needed on ubuntu / regular linux.

File: docs/README.linux [hunk 1]
```diff
@@ -49,7 +49,7 @@ Build-Depends: autoconf, automake, autopoint, autotools-dev, cmake, curl,
   libbluray-dev (>= 0.7.0), libbz2-dev, libcap-dev,
   libcdio-dev, libcec-dev, libcurl4-openssl-dev | libcurl4-gnutls-dev | libcurl-dev,
   libcwiid-dev, libdbus-1-dev, libegl1-mesa-dev, libfontconfig-dev, libfreetype6-dev,
-  libfribidi-dev, libgif-dev (>= 4.1.6), libgl1-mesa-dev | libgl-dev, libglu1-mesa-dev | libglu-dev,
```
Comment (wsnipex) [inline docs/README.linux]: this is still needed

File: docs/README.ubuntu [hunk 2]
```diff
@@ -85,7 +85,7 @@ For Ubuntu (all versions >= 7.04):
       default-jre fp-compiler gawk gdc gettext git-core gperf libasound2-dev libass-dev \
       libbz2-dev libcap-dev libcdio-dev libcurl3 \
       libcurl4-openssl-dev libdbus-1-dev libfontconfig-dev libegl1-mesa-dev libfreetype6-dev \
-      libfribidi-dev libgif-dev libiso9660-dev libjpeg-dev liblzo2-dev \
```
Comment (wsnipex) [inline docs/README.ubuntu]: still needed

File: xbmc/guilib/Makefile.in [hunk 3]
```diff
@@ -103,6 +99,6 @@ include @abs_top_srcdir@/Makefile.include
 -include $(patsubst %.cpp,%.P,$(patsubst %.c,%.P,$(SRCS)))
 
 try : try.o TextureBundle.o DirectXGraphics.o GUIFontTTF.o
-	g++ -o try try.o TextureBundle.o DirectXGraphics.o GraphicContext.o GUIIncludes.o ../utils/CriticalSection.o ../XBVideoConfig.o SkinInfo.o ../linux/CriticalSection.o GUIFontTTF.o GUIFontBase.o GUIFontManager.o GUIFont.o ../utils/XMLUtils.o GUIImage.o GUIControl.o TextureManager.o GUIMessage.o ../utils/SingleLock.o VisibleEffect.o GUIWindowManager.o AnimatedGif.o -lSDL_image -lSDL_gfx -lSDL -llzo -lfreetype
+	g++ -o try try.o TextureBundle.o DirectXGraphics.o GraphicContext.o GUIIncludes.o ../utils/CriticalSection.o ../XBVideoConfig.o SkinInfo.o ../linux/CriticalSection.o GUIFontTTF.o GUIFontBase.o GUIFontManager.o GUIFont.o ../utils/XMLUtils.o GUIImage.o GUIControl.o TextureManager.o GUIMessage.o ../utils/SingleLock.o VisibleEffect.o GUIWindowManager.o -lSDL_image -lSDL_gfx -lSDL -llzo -lfreetype
```
Comment (wsnipex) [inline xbmc/guilib/Makefile.in]: unrelated to this PR, but I think SDL should be removed as well

**Comment by ace20022** [comment]
jenkins build this please

**Comment by ace20022** [comment]
@wsnipex @stefansaraev thx for the help! Good to go?

**Comment by wsnipex** [comment]
looks good to me

**Pull request merged**